earth_enterprise/docs/landing_page/manual/answer/4485225.html

```diff
@@ -141,10 +140,10 @@ <h4 id="enable_the_cutter">Enable the cutter</h4>
   <code># geserveradmin --enable_cutter </code>
   <p>For more information about <code>geserveradmin</code>, please refer to the <strong>Command Line Reference</strong> chapter of the <strong>Reference Guide</strong>.</p>
   <div class="alert">
-    <p><strong>Note about authentication and SSL:</strong> Cutting is not currently supported on globes that require end-user authentication e.g. LDAP. One workaround is to allow unauthenticated access from localhost on your Earth Enterprise Server. (Contact Google Enterprise Support if you need sample Apache directives to enable such a configuration.) Cutting of a globe over HTTPS is supported; however the SSL certificate of the target server will not be verified during cuting.</p>
+    <p><strong>Note about authentication and SSL:</strong> Cutting is not currently supported on globes that require end-user authentication e.g. LDAP. One workaround is to allow unauthenticated access from localhost on your Earth Enterprise Server. (Refer to Apache documentation to enable such a configuration.) Cutting of a globe over HTTPS is supported; however the SSL certificate of the target server will not be verified during cuting.</p>
   </div>
   <h4 id="install_the_google_earth_plugin">Install the Google Earth Plugin</h4>
-  <p>The Google Earth Plugin is used to display the globe from which maps are cut. To install the plugin, visit <code><strong>http://developers.google.com/apis/earth</strong></code> and follow the prompts inside the Google Earth Plugin frame. If you require installers for offline installation, contact Google Enterprise Support.</p>
+  <p>The Google Earth Plugin is used to display the globe from which maps are cut. To install the plugin, visit <code><strong>http://developers.google.com/apis/earth</strong></code> and follow the prompts inside the Google Earth Plugin frame.</p>
```

Comment:
This URL is displayed as code rather than a link.  Also, it's a dead link.  Both of those can be fixed on a different ticket, though.